security/crowdsec/+POST_DEINSTALL.post

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+# Removing the plugin from the web interface will autoremove the dependencies
+# too, and here we have to delete the files in rc.conf.d (because they are
+# generated from templates when the configuration is saved, and the package
+# system did not keep track of them).
+
+# But.. If the plugin is removed from the command line (which does not happen
+# outside of testing conditions), the crowdsec and bouncer services will not be
+# removed. However, since we deleted the files that enabled these services,
+# they will be disabled at the next reboot.
+
+rm -f /etc/rc.conf.d/crowdsec \
+      /etc/rc.conf.d/crowdsec_firewall \
+      /etc/rc.conf.d/oscrowdsec
```

Comment:
I think basically we accept this situation can happen and don't try to account for it as the impact is relatively low.

Reply:
Oh yes, the second paragraph. But I guess I can still remove the files I created.

Reply:
up to you. just wanted to note that side effects will always exist and some are real time consumers that offer almost no direct benefit to users.